### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by Hurleyworks** [InteractiveComputerGraphics/SPlisHSPlasH#157] [Show and tell]
Thanks for making such a high quality project available!  I've been working on a content creation app and found it relatively easy to hook up to SPlisHSPlash because of it's well thought out design.  Here's the DamBreakModel sample rendered in near real time with a path tracer based on NVIDIA"s OptiX7.4

https://youtu.be/0agVO1po58A

**Comment by janbender** [comment]
That looks great!!! 
Thanks for sharing the video.
